### PR TITLE
the pr that lets slimes turn into useless puddles

### PR DESCRIPTION
**Pull request by timothyteakettle**
## About The Pull Request
roundstart slimes get a roundstart action to turn into a puddle / back
as a puddle they're forced to lay down, unable to sprint or enter combat mode, and can pass over tables (contemplating removing this because they're a puddle why do they need to pass tables?), they also drop items in hand and have their arms paralysed while in the form

they also make lots of slime squish noises, including when stepped on

main_large

**testmerge this first**

## Why It's Good For The Game
mainly cosmetic ability that turns slimes into useless puddles because its fun

## Changelog
:cl:
add: roundstart slimes can turn into puddles now
/:cl:

### CODE REVIEW

**Comment by EmeraldSundisk** [comment]
He turned himself into a puddle. Funniest shit I've ever seen.

File: code/modules/mob/living/carbon/human/species_types/jellypeople.dm [hunk 1]
```diff
@@ -641,6 +651,81 @@
...
+			owner.cut_overlays()
+			var/obj/effect/puddle_effect = new puddle_into_effect(get_turf(owner), owner.dir)
+			puddle_effect.color = mutcolor
+			H.Stun(in_transformation_duration, ignore_canstun = TRUE)
+			H.rotate_on_lying = FALSE
+			H.KnockToFloor(TRUE, TRUE, TRUE) //disarms and you cant pickup after
+			ADD_TRAIT(H, TRAIT_PARALYSIS_L_ARM, "SLIME_PUDDLE_PARALYSIS_L_ARM")
+			ADD_TRAIT(H, TRAIT_PARALYSIS_R_ARM, "SLIME_PUDDLE_PARALYSIS_R_ARM")
+			ADD_TRAIT(H, TRAIT_MOBILITY_NOPICKUP, "SLIME_PUDDLE_NO_PICKUP")
+			ADD_TRAIT(H, TRAIT_MOBILITY_NOUSE, "SLIME_PUDDLE_NO_ITEMUSE")
```
Comment (silicons) [inline code/modules/mob/living/carbon/human/species_types/jellypeople.dm]: please use proper trait source in a #define

Comment (timothyteakettle) [reply code/modules/mob/living/carbon/human/species_types/jellypeople.dm]: aight

**Comment by redpepperphoenix** [comment]
neat concept but right away I found that your orientation gets messy. like walking sideways or upside down after turning back.

**Comment by timothyteakettle** [comment]
this has already been fixed there's absolutely no rotation at all now as i rewrote it, it's just the server hasn't been updated

**Pull request merged**